events/2022-january-22/notes-2022-january-22.md

```diff
@@ -0,0 +1,40 @@
+# Notes: Writing image descriptions for documentation
+
+## Notes by section
+
+### Intro talks: Links from slides
+- [Event Code of Conduct](https://jupyter.org/governance/conduct/code_of_conduct.html)
+- [Jupyter.org](https://jupyter.org/)
+- [The WebAIM Million](https://webaim.org/projects/million/)
+- [W3C Web Accessibility Initiative Accessibility Tutorials: Image Concepts](https://www.w3.org/WAI/tutorials/images/)
+- [Alt text hall of fame nominations](https://github.com/Quansight-Labs/jupyter-accessibility-workshops/issues/9)
+- [Event Feedback Survey](https://forms.gle/H6AvxZ718wyGHtxJA)
+- [Jupyter community calendar](https://docs.jupyter.org/en/latest/community/content-community.html#jupyter-community-meetings)
+- [Jupyter Discourse](https://discourse.jupyter.org/c/special-topics/accessibility/29)
+- [GitHub jupyter/accessibility repository](https://github.com/jupyter/accessibility/)
+
+### Breakout sessions
+
+The sessions are just over 30 minutes long. There will be a break in the middle, then we will regroup after the second round.
+
+Sprint topics are
+- contributing image descriptions
+- automated testing for themes
```

Comment:
```suggestion
There are two breakout session groups, each with a different topic:
1. contributing image descriptions
2. automated testing for themes
```

Since you refer to them as group 1 and group 2 below I thought it might help the reader make the connection if you use numerals here too